src/main/java/com/checkmarx/flow/service/EmailService.java

```diff
@@ -13,20 +16,24 @@
 
 import javax.validation.constraints.NotNull;
 import java.beans.ConstructorProperties;
+import java.util.HashMap;
 import java.util.List;
 import java.util.Map;
 
 @Service
 public class EmailService {
 
+    public static final String MESSAGE_KEY = "message";
+    public static final String HEADING_KEY = "heading";
+    public static final String COMPLETED_PROCESSING = "Successfully completed processing for ";
     private static final Logger log = org.slf4j.LoggerFactory.getLogger(EmailService.class);
```

Comment:
@Slf4j annotation of Lombok can be used instead.

Reply:
Done